.env

```
@@ -25,7 +25,7 @@ _APP_OPTIONS_FORCE_HTTPS=disabled
 _APP_OPTIONS_ROUTER_FORCE_HTTPS=disabled
 _APP_OPENSSL_KEY_V1=your-secret-key
 _APP_DNS=172.16.238.100 # CoreDNS
-_APP_DOMAIN=appwrite.test
+_APP_DOMAIN=appwritelocal.tunnel.matejbaco.eu
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_DOMAIN key should go before the _APP_EDITION key

(UnorderedKey)